src/db/oracledb/persons-dao.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+
+import { getConnection } from './connection';
+import { contrib } from './contrib/contrib';
+import { serializePerson } from '../../serializers/persons-serializer';
+
+/**
+ * Queries data source for raw person data and passes it to the serializer
+ *
+ * @param {object} binds object
+ * @returns {Promise<object>} Serialized person resource from person-serializer
+ */
+const getPersonById = async (binds) => {
+  const connection = await getConnection();
+  try {
+    const { rows } = await connection.execute(contrib.getPersonById(), binds);
+
+    if (rows.length > 1) {
+      throw new Error('Expect a single object but got multiple results.');
+    } else if (_.isEmpty(rows)) {
+      return null;
```

Comment:
Can you return `undefined` here for the consistency with other APIs? https://github.com/osu-mist/terms-api/blob/6fc215226eb3b2508b7628ba73d2a748e0125817/src/api/v1/db/oracledb/terms-dao.js#L102